mathics/builtin/fileformats/htmlformat.py

```diff
@@ -165,6 +165,10 @@ def _parse(self, text):
 
 class HTMLGetString(_Get):
     """
+    <dl>
+    <dt>'HTML`Parser`HTMLGetString["string"]'
+    <dd> parses HTML code contained in "string".
+    </dl>
     #> Head[HTML`Parser`HTMLGetString["<a></a>"]]
```

Comment:
Sometime down the line it would be nice to replace this with a unit test - always more to do.

Reply:
Yep, this is for another round, once all the builtins have a properly formatted docstring.